cypress/integration/cucumber/step_definitions/refapp-3.x/01-login/login.js

```diff
@@ -0,0 +1,24 @@
+import {Given} from 'cypress-cucumber-preprocessor/steps';
+
+Given('user arrives at the login page', () => {
+    cy.task('getProperty', 'webapp.url').then(baseUrl => {
+        cy.visit(`${baseUrl}/login`);
+    });
+})
+
+When('the user logs in with {string} and {string}', (username, password) => {
+    cy.get('#username').type(username);
+    cy.contains('Continue').click();
+    cy.get('#password').type(password);
+    cy.contains('Log in').click();
+    cy.wait(3000);
```

Comment:
any reason why you want to delay here?

Reply:
Actually, it takes a long time to load. But later on, I figured out a way to increase the waiting time (by updating the cypress json file). I'll update the PR

Reply:
fixed.